Inputs section of tutorial doc (#188)
ihmeuw/easylink: docs/source/user_guide/tutorials/getting_started.rst

cluster management system which EasyLink can interface with to schedule and run the steps of a pipeline using the resources of a computing cluster. This means that instead of 
running all pipeline steps in your local computing environment, each step can be run with the additional resources of a separate compute node.

To run the pipeline using slurm, we will pass :download:`environment_slurm.yaml <environment_slurm.yaml>` 

> to the ``environment`` command line parameter. Download the file to the directory you will run ``easylink`` 

Reviewer: ```suggestion
to the ``--computing-environment`` command line parameter. Download the file to the directory you will run ``easylink`` 
```

> from -- I downloaded it to the root ``easylink`` directory. The YAML looks like this:

Reviewer: ```suggestion
in -- I downloaded it to the root ``easylink`` directory. The YAML looks like this:
```

faster than ``local``, or make it *possible* to run the pipeline when it wouldn't be otherwise 
(in the case where the local environment doesn't have sufficient resources to run the pipeline).

Input data
==========
The ``--input-data`` (``-i``) argument to ``easylink run`` accepts a YAML file specifying a list 
of paths to files or directories containing input data to be used by the pipeline. 

> When we ran our first pipeline, ``common/pipeline.yaml``, above, we passed 

Reviewer: ```suggestion
When we ran our first pipeline, ``tests/specifications/common/pipeline.yaml``, above, we passed 
```

``tests/specifications/common/input_data.yaml`` 
as this YAML file, shown below::

   input_file_1: /mnt/team/simulation_science/priv/engineering/er_ecosystem/sample_data/dummy/input_file_1.parquet
   input_file_2: /mnt/team/simulation_science/priv/engineering/er_ecosystem/sample_data/dummy/input_file_2.parquet
   input_file_3: /mnt/team/simulation_science/priv/engineering/er_ecosystem/sample_data/dummy/input_file_3.parquet

Let's try passing a different input data specification YAML file, 
:download:`input_data.yaml <input_data.yaml>`, which looks like this::

   input_file_1: input_file_1.parquet
   input_file_2: input_file_2.parquet
   input_file_3: input_file_3.parquet

> Download the file to the directory you will run easylink from, and then download the three input 
> parquets, :download:`input_file_1.parquet <input_file_1.parquet>`, :download:`input_file_2.parquet <input_file_2.parquet>` 
> and :download:`input_file_3.parquet <input_file_3.parquet>` to the same directory. In this case 

Reviewer: ```suggestion
Download the file to the directory you will run easylink in, and then download the three input 
Parquet files, :download:`input_file_1.parquet <input_file_1.parquet>`, :download:`input_file_2.parquet <input_file_2.parquet>` 
and :download:`input_file_3.parquet <input_file_3.parquet>` to the same directory. In this case 
```

They have 100 rows each instead of 10000, the ``bar`` column has a different set of values 
for each file, and the ``counter`` in each file starts at a different value.

Let's run the same pipeline as before, but with this new input data YAML.

.. code-block:: console
   
   $ easylink run -p tests/specifications/common/pipeline.yaml -i input_data.yaml -e tests/specifications/common/environment_local.yaml
   2025-05-01 08:05:01.123 | 0:00:02.781384 | run:158 - Running pipeline
   2025-05-01 08:05:01.123 | 0:00:02.781776 | run:160 - Results directory: /mnt/share/homes/tylerdy/easylink/results/2025_05_01_08_05_01
   2025-05-01 08:05:04.498 | 0:00:06.156166 | main:115 - Running Snakemake
   [Thu May  1 08:05:05 2025]
   Job 9: Validating step_4_python_pandas input slot step_4_secondary_input
   Reason: Missing output files: input_validations/step_4_python_pandas/step_4_secondary_input_validator
   ...
   [Thu May  1 08:05:32 2025]
   Job 0: Grabbing final output
   Reason: Missing output files: result.parquet; Input files updated by another job: intermediate/step_4_python_pandas/result.parquet, input_validations/final_validator

> As expected, the ``results.parqet`` has 600 rows (as opposed to 60k with the old input YAML) 

Reviewer: ```suggestion
As expected, the ``results.parquet`` has 600 rows (as opposed to 60k with the old input YAML) 
```

and the range of ``bar`` and ``counter``  values are consistent 
with our new input files. As before, the transformation of the data is specific to the development schema and will 

> change.

Reviewer: Can you include a `pqprint` that shows this?
